daprdocs/content/en/developing-applications/building-blocks/pubsub/pubsub-message-ttl.md

```diff
@@ -0,0 +1,37 @@
+---
+type: docs
+title: "Pub/Sub Message's Time-To-Live"
+linkTitle: "Pub/Sub Message's TTL"
+weight: 5000
+description: "Use time-to-live in Pub/Sub messages." 
+---
+
+## Introduction
+
+In Kafka, time-to-live is configured in the topic via `retention.ms` as per [documentation](https://kafka.apache.org/documentation/#topicconfigs_retention.ms). This setup is transparent to Dapr.
+
+On the other hand, Azure Service Bus supports [entity level time-to-live]((https://docs.microsoft.com/en-us/azure/service-bus-messaging/message-expiration)). It means that messages have a default time-to-live but also can be set with a shorter timespan at publishing time.
+
+Dapr enables per message time-to-live for all Pub/Sub components, falling back automatically to native support from the component when applicable. It means that applications can set time-to-live per message and subscribers will not receive those messages after expiration.
```

Comment:
This might be a better start to this doc as it provides an overview. Maybe move this up before the Kafka line?

Reply:
Done.